src/shared/modules/discovery/discoveryDuck.ts

```diff
@@ -133,21 +145,34 @@ export const discoveryOnStartupEpic = (some$: any, store: any) => {
     .ofType(APP_START)
     .map((action: any) => {
       if (!action.url) return action
+      const { searchParams } = new URL(action.url)
+
       const passedURL =
-        getUrlParamValue('dbms', action.url) ||
```

Comment:
Lets make sure that the regexp stuff from `getUrlParamValue` is handled by `URL` ...